funnel/views/membership.py

```diff
@@ -316,13 +316,22 @@ class ProjectMembershipView(ProjectViewMixin, UrlChangeCheck, UrlForView, ModelV
     @route('', methods=['GET', 'POST'])
     @render_with('project_membership.html.jinja2')
     def crew(self):
-        return {
-            'project': self.obj,
-            'memberships': [
-                membership.current_access(datasets=('without_parent', 'related'))
-                for membership in self.obj.active_crew_memberships
-            ],
-        }
+        project = self.obj
+        memberships = [
+            membership.current_access(datasets=('without_parent', 'related'))
+            for membership in self.obj.active_crew_memberships
+        ]
+        if request_is_xhr():
+            return jsonify(
+                {
+                    'html': render_template(
+                        'project_membership.html.jinja2',
+                        project=project,
+                        memberships=memberships,
+                    )
+                }
+            )
```

Comment:
This should be given to `render_with`. It can be told to render for different targets.